### PR TITLE
Reduce unwraps

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**
#383

### CODE REVIEW

File: src/network.rs [hunk 1]
```diff
@@ -155,14 +155,17 @@ pub fn hook(path: &'static str, c_daemon_mutex: Arc<Mutex<Daemon>>) {
...
         // Read the response
         let mut reader = BufReader::new(&stream);
         let mut line = String::new();
-        reader.read_line(&mut line).unwrap();
+        reader.read_line(&mut line).expect("Could not read line");
         println!("(debug not for production) Response: {}", line);
-        stream.shutdown(std::net::Shutdown::Both).unwrap();
+        stream
+            .shutdown(std::net::Shutdown::Both)
+            .expect("Coult not shutdown stream");
```
Comment (Camerooooon) [inline src/network.rs:168]: Spelling

**Review by Camerooooon — CHANGES_REQUESTED** [review]

File: src/network.rs (hunk 1 above)
Comment (JakeRoggenbuck) [reply src/network.rs]: Thanks!

File: src/config.rs [hunk 2]
```diff
@@ -118,21 +124,21 @@ impl SafeFillConfig for SafeConfig {
         // will be done to modify base.
         let mut base = default_config();
 
-        if self.powersave_under.is_some() {
-            base.powersave_under = self.powersave_under.unwrap();
+        if let Some(pu) = self.powersave_under {
+            base.powersave_under = pu;
         }
```
Comment (Camerooooon) [inline src/config.rs:129]: Can you explain to me how this is safer? Who does wrapping it in a Some work??

**Review by Camerooooon — APPROVED** [review]
Everything looks good!

File: src/config.rs (hunk 2 above)
Comment (JakeRoggenbuck) [reply src/config.rs:129]: It is a syntax that defines `pu` only if `self.powersave_under` is Some as the unwrapped value of powersave_under. This is similar to the previous code but is easier to understand and without the unwrap written out by the user.

**Pull request merged**